tests/ImageSharp.Tests/Formats/Tiff/BigTiffMetadataTests.cs

```diff
@@ -0,0 +1,260 @@
+                { new ExifTag<float>((ExifTagValue)0xdd02), (ExifDataType.SingleFloat, 2.345f) },
+                { new ExifTag<double[]>((ExifTagValue)0xdd03), (ExifDataType.DoubleFloat, new double[] { 4.5, 6.7 }) },
+                { new ExifTag<double>((ExifTagValue)0xdd04), (ExifDataType.DoubleFloat, 8.903) },
+                { new ExifTag<sbyte>((ExifTagValue)0xdd05), (ExifDataType.SignedByte, (sbyte)-3) },
+                { new ExifTag<sbyte[]>((ExifTagValue)0xdd06), (ExifDataType.SignedByte, new sbyte[] { -3, 0, 5 }) },
+                { new ExifTag<int[]>((ExifTagValue)0xdd07), (ExifDataType.SignedLong, new int[] { int.MinValue, 1, int.MaxValue }) },
+                { new ExifTag<uint[]>((ExifTagValue)0xdd08), (ExifDataType.Long, new uint[] { 0, 1, uint.MaxValue }) },
+                { new ExifTag<short>((ExifTagValue)0xdd09), (ExifDataType.SignedShort, (short)-1234) },
+                { new ExifTag<ushort>((ExifTagValue)0xdd10), (ExifDataType.Short, (ushort)1234) },
+            };
+
+            // arrange
+            var values = new List<IExifValue>();
+            foreach (KeyValuePair<ExifTag, (ExifDataType DataType, object Value)> tag in testTags)
+            {
+                ExifValue newExifValue = ExifValues.Create((ExifTagValue)(ushort)tag.Key, tag.Value.DataType, tag.Value.Value is Array);
+
+                Assert.True(newExifValue.TrySetValue(tag.Value.Value));
+                values.Add(newExifValue);
+            }
+
+            input.Frames.RootFrame.Metadata.ExifProfile = new ExifProfile(values, Array.Empty<ExifTag>());
+
+            // act
+            var encoder = new TiffEncoder();
+            using var memStream = new MemoryStream();
+            input.Save(memStream, encoder);
+
+            // assert
+            memStream.Position = 0;
+            using var output = Image.Load<Rgba32>(memStream);
+            ImageFrameMetadata loadedFrameMetadata = output.Frames.RootFrame.Metadata;
+            foreach (KeyValuePair<ExifTag, (ExifDataType DataType, object Value)> tag in testTags)
+            {
+                IExifValue exifValue = loadedFrameMetadata.ExifProfile.GetValueInternal(tag.Key);
+                Assert.NotNull(exifValue);
+                object value = exifValue.GetValue();
+
+                Assert.Equal(tag.Value.DataType, exifValue.DataType);
+                {
+                    Assert.Equal(value, tag.Value.Value);
+                }
+            }
+        }
+
+        [Fact]
+        public void NotCoveredTags64bit()
+        {
+            var testTags = new Dictionary<ExifTag, (ExifDataType DataType, object Value)>
+            {
+                { new ExifTag<ulong>((ExifTagValue)0xdd11), (ExifDataType.Long8, ulong.MaxValue) },
+                { new ExifTag<long>((ExifTagValue)0xdd12), (ExifDataType.SignedLong8, long.MaxValue) },
+                //// WriteIfdTags64Bit: arrays aren't support
```

Comment:
How come? In the spec or our code?

Reply:
Our code. No test files for coverage (64-bit arrays).